.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
i think we should add `.idea` to a `.gitignore` file so this wont mess with anyone else's configuration